cluster/manifests/z-karpenter/01-rbac.yaml

```diff
@@ -146,7 +146,7 @@ rules:
       - config-logging
   - apiGroups: ["coordination.k8s.io"]
     resources: ["leases"]
-    verbs: ["patch", "update"]
```

Comment:
This seems suspicious. I would think it's needed to patch/update for leader election basically the current leader saying: "I'm still here to be the leader".

Reply:
Yes, this doesn't work. See this in teapot:

```
Failed to update lock: leases.coordination.k8s.io "karpenter-leader-election" is forbidden: User "system:serviceaccount:kube-system:karpenter" cannot update resource "leases" in API group "coordination.k8s.io" in the namespace "kube-system": access undecided system:serviceaccount:kube-system:karpenter/[system:serviceaccounts system:serviceaccounts:kube-system system:authenticated]
```

Reply:
Ah, interesting. I'll add the verbs again.